requirements.txt

```diff
@@ -1,5 +1,11 @@
 lxml
 requests
+boto3==1.9.44
+boto==2.49.0
+botocore==1.12.44         # via boto3, s3transfer
+json5==0.6.1              # via anymarkup
+semantic-version==2.8.4
+unified-range==0.1.0
```

Comment:
This causes Test Fail. Why we need it?

Reply:
its a pkg used for getting semantic versions for maven and python. Their latest version is suddenly causing problems. need to downgrade the version and am testing